dockerfile/rust/native.Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM rust:1-bullseye AS build-env
 
 RUN rustup component add rustfmt
 
-RUN apt install -y libssl1.1:amd64 libssl-dev:amd64 openssl:amd64 libclang-dev clang cmake protobuf-compiler
+RUN apt install -y libssl1.1 libssl-dev:amd64 openssl libclang-dev clang cmake protobuf-compiler
```

Comment:
Still have one remaining `:amd64` in there. Also, needed a `apt update &&` at the beginning.